src/components/builds/DebuggingInformation.tsx

```diff
@@ -71,18 +71,25 @@ function DebuggingInformation(props: Props) {
     );
   }
 
-  return (
-    <Card>
-      <CardContent>
-        <Typography variant="h6">Debugging Information</Typography>
-        <div className={classes.gapped}>YAML configuration</div>
-        {generateTable(build.parsingResult.rawYamlConfig.split('\n'))}
+  const starlarkInformation =
+    build.parsingResult.rawStarlarkConfig === '' ? null : (
+      <div>
```

Comment:
https://stackoverflow.com/questions/42122121/return-multiple-react-elements-in-a-method-without-a-wrapper-element/44337553#44337553

Reply:
Actually you're right: the SO answer mentions an article that explains how to avoid `<div>`-wrapping in React 16 (https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html), which we're already on.

Fixed in https://github.com/cirruslabs/cirrus-ci-web/pull/369/commits/2d39100a1148201e08398b04363c6690a986feb1.